### PR TITLE
[pt-br] Translation files from tsconfig-reference

### PR DESCRIPTION
**Pull request by cataua**
My first contribution, I hope it's helpful and I made all proceed correctly. @khaosdoctor,  @alvarocamillont, @danilofuchs e @orta 

The files are:
Command_line_Options_6171.md
Experimental_Options_6177.md
Module_Resolution_Options_6174.md
Project_Files_0.md
Source_Map_Options_6175.md
Strict_Type_Checking_Options_6173.md
Watch_Options_999.md

### CODE REVIEW

File: packages/tsconfig-reference/copy/pt/categories/Experimental_Options_6177.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+display: "Experimental"
+---
+
+TypeScript se esforça para incluir apenas funcionalidades que a adição está confirmada na linguagem Javascript.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/categories/Experimental_Options_6177.md]:
```suggestion
TypeScript se esforça para incluir apenas funcionalidades cujas adições estão confirmadas na linguagem Javascript.
```

File: packages/tsconfig-reference/copy/pt/categories/Experimental_Options_6177.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Existem casos que a funcionalidade é atraente o suficiente para ser uma exceção a essa regra, e nesses casos possuem a sinalização do compilador de experimental.
+É possível que uma versão dessas funcionalidades seja diferente quando/se forem adicionadas a linguagem Javascript e, portante, são considerados de risco.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/categories/Experimental_Options_6177.md]:
```suggestion
É possível que uma versão dessas funcionalidades seja diferente quando/se forem adicionadas a linguagem Javascript e, portanto, são consideradas de risco.
```

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+display: "Verificações rígidas"
+---
+
+Recomendamos utilizar [compiler option `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
Recomendamos utilizar [a opção `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
```

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+TypeScript suporta um amplo espectro dos padrões do Javascript e padrões para permitir bastante flexibilidade em acomodar estes estilos.
+Freqüentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
Frequentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
```

File: packages/tsconfig-reference/copy/pt/categories/Watch_Options_999.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+display: "Watch Options"
+---
+
+O TypeScript 3.8 lançou uma nova estratégia para monitorar as pastas, que é crucial para obter as mudanças no `node_modules`.
+
+Nos sistemas operacionais como Linux, o Typescript instala monitores de diretórios (ao contrário dos monitores de arquivos) no `node_modules` e muitos dos seus sub-diretórios para detectar mudanças nas suas dependências.
+Isto é devido a quantidade de monitores de arquivos disponíveis que é eclipsado pelo número de arquivos no `node_modules`, enquanto há muito poucos diretórios para rastrear.
```
Comment (khaosdoctor) [inline packages/tsconfig-reference/copy/pt/categories/Watch_Options_999.md:8]:
```suggestion
Isto é devido a quantidade de monitores de arquivos disponíveis, que são eclipsados pelo número de arquivos no `node_modules`, enquanto existem poucos diretórios para rastrear.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Small changes on files

**Review by khaosdoctor — APPROVED** [review]
lgtm

**Comment by khaosdoctor** [comment]
@orta can you resolve this conflict? I don't have permissions

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+Recomendamos utilizar [a opção `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
+
+TypeScript suporta um amplo espectro dos padrões do Javascript e padrões para permitir bastante flexibilidade em acomodar estes estilos.
+Frequentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
+
+Devido a variedade do Javascript suportado, a atualização para uma nova versão pode revelar dois tipos de erros:
+
+- Erros que já existem no seu código base, que o Typescript revelou pela linguagem melhorou seu entendimento do Javascript.
+- Um novo conunto de erros que abordam um novo escopo de problema.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
- Um novo conjunto de erros que abordam um novo escopo de problema.
```

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+Recomendamos utilizar [a opção `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
+
+TypeScript suporta um amplo espectro dos padrões do Javascript e padrões para permitir bastante flexibilidade em acomodar estes estilos.
+Frequentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
+
+Devido a variedade do Javascript suportado, a atualização para uma nova versão pode revelar dois tipos de erros:
+
+- Erros que já existem no seu código base, que o Typescript revelou pela linguagem melhorou seu entendimento do Javascript.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
- Erros que já existem no seu código base, que o Typescript passa a revelar, dado um melhor entendimento de Javascript.
```
Reduzir ambiguidade

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+---
+display: "Verificações rígidas"
+---
+
+Recomendamos utilizar [a opção `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
+
+TypeScript suporta um amplo espectro dos padrões do Javascript e padrões para permitir bastante flexibilidade em acomodar estes estilos.
+Frequentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
+
+Devido a variedade do Javascript suportado, a atualização para uma nova versão pode revelar dois tipos de erros:
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
Devido a variedade do Javascript suportado, a atualização para uma nova versão do Typescript pode revelar dois tipos de erros:
```
Reduzir ambiguidade

File: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+- Erros que já existem no seu código base, que o Typescript revelou pela linguagem melhorou seu entendimento do Javascript.
+- Um novo conunto de erros que abordam um novo escopo de problema.
+
+O TypeScript irá adcionar um sinalizador de compilador para o último conjunto de erros, e por padrão estes não estão habilitados.
```
Comment (danilofuchs) [inline packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
O TypeScript irá adicionar um sinalizador de compilador para o último conjunto de erros, e por padrão estes não estão habilitados.
```

**Review by danilofuchs — CHANGES_REQUESTED** [review]

**Review by danilofuchs — APPROVED** [review]
LGTM

**Comment by orta** [comment]
Ah, close

```
Looking at the pull_request_review from danilofuchs in '[pt-br] Translation files from tsconfig-reference' to see if we can merge

Changed files: 
 - /packages/tsconfig-reference/copy/pt/categories/Command_line_Options_6171.md
 - /packages/tsconfig-reference/copy/pt/categories/Experimental_Options_6177.md
 - /packages/tsconfig-reference/copy/pt/categories/Module_Resolution_Options_6174.md
 - /packages/tsconfig-reference/copy/pt/categories/Project_Files_0.md
 - /packages/tsconfig-reference/copy/pt/categories/Source_Map_Options_6175.md
 - /packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md
 - /packages/tsconfig-reference/copy/pt/categories/Watch_Options_999.md
 - /packages/typescriptlang-org/src/lib/release-info.json

@danilofuchs does not have access to merge 
 - /packages/typescriptlang-org/src/lib/release-info.json
```

**Comment by danilofuchs** [comment]
I cannot merge since there are changes in `release-info.json`. I've seen this file being updated quite frequently in translation PRs. Is this expected, @orta?

**Comment by orta** [comment]
Yeah, maybe I should just gitignore it at this point

**Comment by danilofuchs** [comment]
> Ah, close

Hahaha so close, indeed. Next time it's going to work flawlessly, I'm sure!

**Comment by khaosdoctor** [comment]
@cataua Please update the spreadsheet on #233 so we know you finished :)

**Comment by cataua** [comment]
> @cataua Please update the spreadsheet on #233 so we know you finished :)

Done, @khaosdoctor !

**Pull request merged**